docs/configuration.rst

```diff
@@ -166,6 +166,19 @@ of volume options:
         hostpath: /host/foo
         options: ro,cached
 
+The keys used in volume mappings can contain environment variables **that are
```

Comment:
It's actually not just the keys (container path), but also the host path, which can contain environment vars (as shown in the example)

Reply:
Opened #214 to fix